UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.SubscriptionService/SubscriptionServiceJob.cs

```diff
@@ -116,5 +121,35 @@ public async Task ProcessQueueMessage([QueueTrigger("%SubscriptionStorageConfigu
             _logger.LogInformation(EventIds.CreateSubscriptionRequestCompleted.ToEventId(),
                     "Subscription provisioning request Completed for SubscriptionId:{SubscriptionId} and _D365-Correlation-ID:{correlationId} and _X-Correlation-ID:{CorrelationId}", subscriptionMessage.SubscriptionId, subscriptionMessage.D365CorrelationId, subscriptionMessage.CorrelationId);
         }
+
+        public async Task ProcessBlobTrigger([BlobTrigger("%SubscriptionStorageConfiguration:StorageContainerName%/{filePath}")] Stream myBlob, string filePath)
+        {
+            string subscriptionId = GetSubscriptionIdName(filePath);
```

Comment:
Please rename method as below:
```suggestion
            string subscriptionId = GetSubscriptionId(filePath);
```

Reply:
Done required changes.